src/types/Tab.tsx

```diff
@@ -0,0 +1,5 @@
+export interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
```

Comment:
there is no JSX inside this file, no need to specify .tsx extension, it should be .ts.